packages/web-components/src/components/rux-pop-up/rux-pop-up.tsx

```diff
@@ -59,6 +59,11 @@ export class RuxPopUp {
      */
     @Prop({ reflect: true }) disableAutoUpdate: boolean = false
 
+    /**
+     * Turns on floatin-ui's autoUpdate animationFrame option to watch for trigger movements and replace the popup if movement is detected. Defaults to false.
```

Comment:
```suggestion
     * watches for trigger movements and replace the popup if movement is detected. 
```

I think we should avoid calling out floatingui explicitly. as far as our end consumer is concerned, the only API they need to worry about is our rux pop up API. This also gives us the flexibility to replace floating-ui under the hood if we ever decided there was a strong reason to. 

also we dont need to specify here that it defaults to false. stencil will parse that out and storybook will show it under the default column in the API table